plugins/outputs/stomp/stomp.go

```diff
@@ -0,0 +1,120 @@
+package stomp
+
+import (
+	ts "crypto/tls"
+	"net"
+
+	"github.com/go-stomp/stomp"
+
+	"github.com/influxdata/telegraf"
+	commontls "github.com/influxdata/telegraf/plugins/common/tls"
+	"github.com/influxdata/telegraf/plugins/outputs"
+	"github.com/influxdata/telegraf/plugins/serializers"
+)
+
+//STOMP ...
+type STOMP struct {
+	Host      string `toml:"host"`
+	Username  string `toml:"username,omitempty"`
+	Password  string `toml:"password,omitempty"`
+	QueueName string `toml:"queueName"`
+	commontls.ClientConfig
+	conn      *ts.Conn
+	netConn   net.Conn
+	stomp     *stomp.Conn
+	serialize serializers.Serializer
+	Log       telegraf.Logger `toml:"-"`
```

Comment:
We usually group the fields like
```suggestion
	Host      string `toml:"host"`
	Username  string `toml:"username,omitempty"`
	Password  string `toml:"password,omitempty"`
	QueueName string `toml:"queueName"`
	Log       telegraf.Logger `toml:"-"`

	commontls.ClientConfig
	
	conn      *ts.Conn
	netConn   net.Conn
	stomp     *stomp.Conn
	
	serialize serializers.Serializer
```